Configuration/TypoScript/Library/page.gtm.setupts

```diff
@@ -0,0 +1,21 @@
+# Google Analytics track pageview and add script for event tracking
+[globalString = LIT:{$themes.configuration.tracking.ga_gtmid} = /(GTM)-[a-zA-Z0-9]+/]
```

Comment:
Can we use an implementation without TYPO3 conditions.

These problem can be solved with an TypoScript `if`.

Every Condition is a performance problem.
Every condition is evaluated at every request.
An `if` is only evaluated once and the result is cached.

Reply:
Can you provide an example how you want to implement it?

Reply:
```
page.headerData.17895 = TEXT
page.headerData.17895 {
	value = ...
	if.isTrue = {$themes.configuration.tracking.ga_gtmid}
}

page.3 = TEXT
page.3 {
	value = ...
	if.isTrue = {$themes.configuration.tracking.ga_gtmid}
}
```

See https://docs.typo3.org/typo3cms/TyposcriptReference/Functions/If/Index.html
The test with regex is not really needed.